modules/sdk-lib-mpc/src/tss/ecdsa/noSmallFactorsProof.ts

```diff
@@ -0,0 +1,119 @@
+import { createHash } from 'crypto';
```

Comment:
Can you please link the paper/reference used for this implementation at the top of this file?

Reply:
done